src/steps/10-FinancialDetails/AppropriationOfFunds.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <v-container class="container-max-width mb-7" fluid>
+    <v-row>
+      <v-col class="col-12">
+        <h1 class="page-header">Let’s find out more about your appropriation of funds</h1>
+        <div class="copy-max-width">
+          <p>
+            Based on what you told us about your exception to fair opportunity, we need to know
+            more about where these funds will be coming from. Your responses below, along with
+            your total estimated value generated in the Requirements Cost Estimate section,
+            will be included in your Justification and Approval (J&A).
+          </p>
+          <div class="max-width-740">
+            <ATATRadioGroup id="fundTypes" 
+              legend="What type of funds are planned for this acquisition?" 
+              :items="fundTypes" 
+              :value.sync="selectedFundType"
+              :rules="[
+                  $validators.required('Please select type of funds.'),
+                ]" 
+              class="mb-10" 
+              >
+            </ATATRadioGroup>
+            <ATATRadioGroup id="ContactAffiliation" 
+              legend="What fiscal year funds are planned for this acquisition?" 
+              :items="fiscalYearFunds" 
+              :value.sync="selectedFiscalYear"
+              class="mb-10" 
+              :rules="[$validators.required('Please select an option.')]"
+              >
+            </ATATRadioGroup>
+          </div>
+        </div>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+<script lang="ts">
+import Vue from "vue";
+
+import { Component, Mixins } from "vue-property-decorator";
+import ATATRadioGroup from "../../components/ATATRadioGroup.vue";
+import SaveOnLeave from "@/mixins/saveOnLeave";
+import { RadioButton } from "types/Global";
+import { FundingRequestDTO } from "@/api/models";
+import FinancialDetails from "@/store/financialDetails";
+import { hasChanges } from "@/helpers";
+import _ from "lodash";
+
+@Component({
+  components: {
+    ATATRadioGroup,
+  },
+})
+
+export default class AppropriationOfFunds extends Mixins(SaveOnLeave) {
+  private fundingRequest: FundingRequestDTO ={};
+  private selectedFundType: "" | "O_M" | "RDT_E" | "PROCUREMENT" = "";
+  private fundTypes: RadioButton[] = [
+    {
+      id: "O&M",
+      label: "Operations & Maintenance",
```

Comment:
Add abbreviations in parentheses to the first 2 labels as in the figma

<img width="1080" alt="Screen Shot 2023-05-08 at 3 57 51 PM" src="https://user-images.githubusercontent.com/90333349/236922074-00ccd18f-8022-4d63-a3ff-b7deb675ff4d.png">